0008-interledger-quoting-protocol/0008-interledger-quoting-protocol.md

```diff
@@ -33,108 +31,90 @@ The sending and receiving ledger are different ledgers. Otherwise, there should
+### IlqpByDestinationRequest
 
-| Field                         | Type        | Description |
-|:------------------------------|:------------|:------------|
-| `source_connector_account`    | ILP Address | The address of the connector's account in the source ledger where it should receive the first transfer. |
-| `source_ledger`               | ILP Address | The address of the ILP-enabled ledger where the source account is. |
-| `source_amount`               | String      | Integer number amount of currency the connector's account should receive in the source ledger. |
-| `source_expiry_duration`      | String      | Integer number of milliseconds between when the payment in the source ledger is prepared and when it must be executed. |
-| `destination_ledger`          | ILP Address | The address of the ILP-enabled ledger where the destination account is. Equivalent to the
-[`appliesToPrefix`](https://github.com/interledger/rfcs/blob/9664732/asn1/InterledgerQuotingProtocol.asn#L36-L42) field which will possibly be used in the future. |
-| `destination_amount`          | String      | Integer number amount of currency that should be received by the destination account in the destination ledger. |
-| `destination_expiry_duration` | String      | Integer number of milliseconds between when the payment in the destination ledger is prepared and when it must be executed. |
-| `liquidity_curve`             | Number[][]  | Piece-wise linear function that was used to calculate this response, and that applies to the entire destination ledger. |
+| Field | Type | Short Description |
+|:--|:--|:--|
+| destinationAccount | Address | Address corresponding to the destination account |
+| destinationAmount | UInt64 | Amount that will arrive at the receiver |
+| destinationHoldDuration | UInt32 | How much time the receiver needs to fulfill the payment (in milliseconds) |
 
-##### Example response (method = `"quote_response"`)
+### IlqpByDestinationResponse
+
+| Field | Type | Short Description |
+|:--|:--|:--|
+| sourceAmount | UInt64 | Amount the sender needs to send, denominated in the asset of the source ledger |
+| sourceHoldDuration | UInt32 | How long the sender should put money on hold (in milliseconds) |
+
+## Example Messages
+##### Example request message
 
 ```js
 {
-  "method": "quote_response",
-  "id": "721e4126-98a1-4974-b35a-8a8f4655f934",
-  "data": {
-    "source_connector_account":"example.eur-ledger.mark",
-    "source_ledger":"example.eur-ledger.",
-    "source_amount":"10025000000",
-    "source_expiry_duration":"6000",
-    "destination_ledger":"example.usd-ledger.",
-    "destination_amount":"9000000000",
-    "destination_expiry_duration":"5",
-    "liquidity_curve":[
-      ["10", "0"],
-      ["100000000000000000", "99799999999999"]
-    ]}
-  }
+  "ledger": "example.eur-ledger.",
+  "from": "example.eur-ledger.alice",
+  "to": "example.usd-ledger.bob",
+  "ilp": "BB8RZXhhbXBsZS5uZXh1cy5ib2IAAAACGHEaAAAAC7gA"
```

Comment:
wait, what happened here? This doesn't match any of the three formats described above?